presto-kafka/src/test/java/io/prestosql/plugin/kafka/TestKafkaIntegrationSmokeTest.java

```diff
@@ -141,6 +141,10 @@ public void testRoundTripAllFormats(RoundTripTestCase testCase)
     private List<RoundTripTestCase> testRoundTripAllFormatsData()
     {
         return ImmutableList.<RoundTripTestCase>builder()
+                .add(new RoundTripTestCase(
```

Comment:
can we write more than single row in the test?

(this actually applies to other formats too)